src/v2/guide/transitions.md

```diff
@@ -569,6 +569,8 @@ By default, this will use the transitions specified for entering and leaving. If
 
 and custom JavaScript hooks:
 
+<p class="tip">Both `appear` attribute and `v-on:appear` hook will cause an appear transition. Compared to `appear` attribute, `v-on:appear` hook has a higher priority. If `v-on:appear` hook is specified, using `:appear="false"` does not cancel the transition.</p>
```

Comment:
This `p.tip` here breaks the flow though. Do you think you can rewrite this content in a way that doesn't require a `p.tip`?

Reply:
Do you think it is okay to keep `p.tip` but move this part below the code snippet?

Reply:
Please do heed An's advice here and rewrite it into the flow of the document. Thanks!

Reply:
I have changed the layout and removed the `p.tip`